app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div id="expenses" class="groups">
+  <% if @expenses.count.zero? %>
+    <p class="center-message"><em>No transactions available for the moment</em></p>
+  <% else %>
+    <% @expenses.each do |expense| %>
+      <%= render expense %>
+    <% end %>
+  <% end %>
+</div>
+
+<div class="add-category">
+  <%= link_to "Add transaction", new_group_expense_path, class: "splash-login add-category-button"%>
+</div>
```

Comment:
- [ ] Good job adding the required functionalities, but please ensure that your design stick to the original guidelines. 
- [ ] The button `Add transaction` should be at the bottom of the page.
- [ ] The icons should also show

### Your Design
![image](https://user-images.githubusercontent.com/43995830/195758604-b528abc8-4eb0-4425-95c8-181c6b78fc73.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/195758485-6d22f620-d8f7-4438-96ec-ae15afa2e3b3.png)

Reply:
Hi, @AnselemOdims!
Thanks for the observation. Now, the icons are shown, and I fixed the position of the button to the bottom. 😁 

<img src="https://user-images.githubusercontent.com/97407840/195899486-15a60cb8-bccf-4a36-bd69-33680a564f3c.png" width="250">